ROADMAP.rst

```diff
@@ -1,21 +1,21 @@
 Roadmap for Ginga
-==================
+=================
 
 This is a roadmap for Ginga development, which lays out
 the long-term plans for this package.
 
 *Note: As a caveat, this is a living document, and
 will evolve as priorities grow and shift.*
 
-Last updated: May 2019
+Last updated: Jun 2021
 
 Short-Term Goals
 ----------------
 
 Things that might happen in the next release or two:
 
-* Dropping Python 2 support (bug fixes to 2.x.y branches until end of 2019).
-* Support ASDF and GWCS.
+* Replace ``astropy_wcs`` with ``astropy_wcs_ape14`` and deprecate the latter
```

Comment:
I removed "Improve support for ASDF and GWCS" completely because I haven't heard any complains about the initial support that we have.